client/my-sites/plugins/plugins-results-header/style.scss

```diff
@@ -10,7 +10,7 @@
 
 		.plugins-results-header__title {
 			@extend .wp-brand-font;
-			font-size: $font-title-medium;
+			font-size: rem(26px); //typography-exception
```

Comment:
@vinimotaa, related to our last call where we talked about unifying designs + Calypso styles. Currently, we have these font sizes on Calypso. Do you think it would be possible for us to reuse one of these here?

<img width="519" alt="image" src="https://user-images.githubusercontent.com/1044309/195707193-92cc6472-a1d0-4b4d-8279-73765be70749.png">

Reply:
```suggestion
			font-size: $font-title-medium;
```

Lets stick with the standard sizing for now and ship the other fixes.